### ISSUE TITLE
expect_equal() failed to capture argument: `tol` 

### ISSUE THREAD

**Issue by chuvanan** [markvanderloo/tinytest#17]
I got an error when trying to use `expect_equal()` even though the equivalent `all.equal()` gave correct result.

```r

expect_equal(
    my_function(binary_actual, predicted),
    target = 0.4861996,
    tol = 10e-8
)
# FALSE

all.equal(my_function(binary_actual, predicted),
          target = 0.4861996,
          tolerance = 10e-8)
# TRUE
```

It seems that the argument `tol` is not used by `expect_equal()`:
```r
expect_equal <- function(current, target, tol = sqrt(.Machine$double.eps), ...){

  check <- all.equal(current, target,...)
  [truncated]
}
```

I think the correct way should be:
```r
expect_equal <- function(current, target, tol = sqrt(.Machine$double.eps), ...){

  check <- all.equal(current, target, tolerance = tol, ...)
  [truncated]
}

```

**Comment by markvanderloo** [comment]
Thank you so much for reporting. You are absolutely corrent. Fixed now as of 0a104187f0bbc0361fd54b209fa23239f91ed5ac.

**Comment by markvanderloo** [comment]
A workaround until this is released is to use `round()` on your `current` and `target` arguments.
